packages/common/src/dto/submission-payload.dto.ts

```diff
@@ -30,4 +32,7 @@ export class SubmissionPayloadDTO {
   @ValidateNested()
   @IsNotEmpty()
   preferencesInformation!: PreferencesInformationDTO;
+
+  @IsOptional()
```

Comment:
Just curious - Why did you add the StatusUpdate DTO to the form submission one instead of having it as a stand alone.